akka-http/src/main/java/akka/http/javadsl/coding/Coder.java

```diff
@@ -25,9 +29,40 @@
         this.underlying = underlying;
     }
 
+    public HttpResponse encodeMessage(HttpResponse message) {
```

Comment:
My Java generic typing Foo could not make it work in a better way, like for example expressing a `T` that extends both `HttpMessage` and `MessageTransformations`.
Comment is valid for the other `encodeMessage` and `decodeMessage` methods.